README.md

```diff
@@ -0,0 +1,34 @@
+# ClipBit
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)

```

Comment:
Add some more icons and stuff. Maybe look if there's a nice looking template. Otherwise, you can commit the minimum stuff and i'll see if I can beautify it or something later

Reply:
sure thing. Let me add